ansible/roles/network/files/wifi-connect.service

```diff
@@ -2,13 +2,13 @@
 Description=Wifi Connect
 Wants=network-online.target
 After=network-online.target
-ConditionPathExists=!/home/pi/.screenly/initialized
 
 [Service]
 WorkingDirectory=/home/pi/screenly
 User=pi
 Type=oneshot
 
+ExecStartPre=/bin/rm -f /home/pi/.screenly/initialized
```

Comment:
Wouldn't this cause the wifi to be resetted at *every* run?

Reply:
> Wouldn't this cause the wifi to be resetted at _every_ run?

No, because the script checks if there is an active point or network connection. After script execute `ExecStartPost=/usr/bin/touch /home/pi/.screenly/initialized`.
It was checked manually, after restart everything works fine.

This part is necessary that the viewer not to skip the `hotspot` page.